Dockerfile

```diff
@@ -25,44 +23,41 @@ RUN apt-get update \
     software-properties-common \
     && rm -rf /var/lib/apt/lists/*
 
+# install solc for py_test if arch is not arm64 because it is not availiable 
+RUN \
+if [ `arch` != "aarch64" ]; \
+then  \
+    add-apt-repository ppa:ethereum/ethereum \
+    && apt-get update \
+    && apt install solc; \
+fi 
 
-# Install LLVM
-RUN curl -SL -o llvm.sh https://apt.llvm.org/llvm.sh && \
-    chmod +x llvm.sh && \
-    ./llvm.sh $LLVM_VERSION && \
-    rm -f llvm.sh
-
-# To get Solidity compiler for python integration tests
 # install standart tools
 RUN add-apt-repository ppa:ethereum/ethereum \
     && apt-get update \
     && apt-get install -y \ 
     clang-format-$LLVM_VERSION \
     clang-tidy-$LLVM_VERSION \
+    llvm-$LLVM_VERSION \
+    golang-go \
     ca-certificates \
     libtool \
     autoconf \
     binutils \
-    cmake=$CMAKE_VERSION \
+    cmake \
+    libsnappy-dev \
+    libzstd-dev \
```

Comment:
why do we need those?

Reply:
it is not needed for me and docker is fine without them right now

Reply:
I added comment that we need this only for arm64 builds